### PR TITLE
Homework nr 5

### PR DESCRIPTION
**Pull request by ArtiomLevitchii**

### CODE REVIEW

File: src/main/java/org/example/homework_nr_5/Weekday.java
Comment (gh3rtzz) [inline src/main/java/org/example/homework_nr_5/Weekday.java:1]: This is somewhat correct but it's very inefficient. Try to use `this` keyword in order to achieve the same result.

Something like
`boolean isWeekDay(){if(this != SATURDAY && this != SUNDAY) return true else return false;}`

File: src/main/java/org.example/homework_nr_1/Main.java
Comment (gh3rtzz) [inline src/main/java/org.example/homework_nr_1/Main.java:1]: Why is this file here?

**Comment by ArtiomLevitchii** [comment]
Fixed, just for my education, can you explain on wednesday, why code is inefficient. TY

**Review by gh3rtzz — APPROVED** [review]
approved
